ls1intum/Artemis
# `Development`: Fix a migration to make it Postgres compatible

Fix #6549: change a previous migration which did not support Postgres.
I removed the old migration, also from `master.xml`
This means:
1. The delete operation will be executed (potentially again) on production system, which is not an issue
2. Adding the foreign key will only be done, if the foreign key does not exist yet

### Checklist
#### General


### Steps for Testing

This can only be tested by system admins.
Deploy the Artemis server to a MySQL and a Postgres system and make sure that no database migration errors happen in the console. In case there are still conversations referencing deleted courses before you deploy, make sure those are deleted now.

### Review Progress


#### Code Review
#### Manual Tests
